expression/builtin_arithmetic_vec.go

```diff
@@ -736,6 +780,10 @@ func (b *builtinArithmeticMultiplyIntSig) vectorized() bool {
 	return true
 }
 
+func (b *builtinArithmeticMultiplyIntSig) nullable(notNullCols []*Column) bool {
+	return true
```

Comment:
if overflow is nullable here, why is `builtinArithmeticPlusIntSig` considered non-nullable (and why does this function call `result.SetNull()` on error while all other doesn't)?

Reply:
I think it is because of the compatibility with mysql.